tools/eval/eval.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import argparse
+
+import numpy as np
+import cv2 as cv
+
+from datasets import DATASETS
+
+if "PYTHONPATH" in os.environ:
+    root_dir = os.environ["PYTHONPATH"]
+else:
+    root_dir = os.path.join("..", "..")
+sys.path.append(root_dir)
+from models import MODELS
+
+parser = argparse.ArgumentParser("Evaluation with OpenCV on different models in the zoo.")
+parser.add_argument("--model", "-m", type=str, required=True, help="model name")
+parser.add_argument("--dataset", "-d", type=str, required=True, help="Dataset name")
+parser.add_argument("--dataset_root", "-dr", type=str, required=True, help="Root directory of given dataset")
+args = parser.parse_args()
+
+models = dict(
+        mobilenetv1=dict(
+            name="MobileNetV1",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_mobilenet/image_classification_mobilenetv1_2022apr.onnx"),
+            topK=5),
+        mobilenetv1_q=dict(
+            name="MobileNetV1",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_mobilenet/image_classification_mobilenetv1_2022apr-int8-quantized.onnx"),
+            topK=5),
+        mobilenetv2=dict(
+            name="MobileNetV2",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_mobilenet/image_classification_mobilenetv2_2022apr.onnx"),
+            topK=5),
+        mobilenetv2_q=dict(
+            name="MobileNetV2",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_mobilenet/image_classification_mobilenetv2_2022apr-int8-quantized.onnx"),
+            topK=5),
+        ppresnet=dict(
+            name="PPResNet",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_ppresnet/image_classification_ppresnet50_2022jan.onnx"),
+            topK=5),
+        ppresnet_q=dict(
+            name="PPResNet",
+            topic="image_classification",
+            modelPath=os.path.join(root_dir, "models/image_classification_ppresnet/image_classification_ppresnet50_2022jan-act_int8-wt_int8-quantized.onnx"),
+            topK=5),
+)
+
+datasets = dict(
+        imagenet=dict(
+            name="ImageNet",
+            topic="image_classification",
+            size=224),
+)
+
+def main(args):
+    # Instantiate model
```

Comment:
At the beginning of the script we can print the model name and test data home name.

Reply:
Such information is printed in `dataset.eval()` like the following:

![image](https://user-images.githubusercontent.com/17219438/177910460-a6068ad0-d847-4759-a660-e8fceea4e4c2.png)